src/wwwroot/app/services/VacancyService.js

```diff
@@ -132,7 +141,11 @@ function _fillThesauruses(vacancy) {
    }, {});
 
    vacancy.languageSkill = vacancy.languageSkill || {};
+   _ThesaurusService.getThesaurusTopic('languageLevels', vacancy.languageSkill.languageLevel)
```

Comment:
curried set was used for this: 
   _ThesaurusService.getThesaurusTopic('languageLevels', vacancy.languageSkill.languageLevel).then(curriedSet(vacancy, 'languageSkill.languageLevelObj'));